index.html

```diff
@@ -0,0 +1,448 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
+        <i class="fas fa-times close-btn"></i>
+      </div>
+
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href=".projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href=".contact"
+              >Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div id="intro">
```

Comment:
[OPTIONAL] Nice implementation of the footer tag. You can also use a section tag instead of a div tag to compose the different sections inside the main eg portfolio section, about me section, and contact me section. This really beneficial for those who needs accessibility assistance like the screen readers etc.